src/main/webapp/WEB-INF/tags/student/studentProfileDiv.tag

```diff
@@ -0,0 +1,82 @@
+<%@ tag description="studentProfile - Edit profile div" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.ui.template.StudentProfileEditDiv" %>
```

Comment:
I believe you do not need to import a class unless using constants and such from it.
Also, I recommend naming it after just the component rather than calling it a "div".